cxx/pclean/schema_helper.cc

```diff
@@ -91,7 +91,7 @@ T_schema PCleanSchemaHelper::make_hirm_schema() {
       if (const ScalarVar* dv = std::get_if<ScalarVar>(&(v.spec))) {
         std::vector<std::string> domains;
         domains.push_back(c.name);
-        for (const std::string& sc : get_source_classes(c.name)) {
+        for (const std::string& sc : get_ancestor_classes(c.name)) {
```

Comment:
Can we delete `get_source_classes`?

Reply:
Sure.  Done.

Reply:
It looks like `schema_test` still calls it (which confuses me if the tests pass)